README.md

```diff
@@ -202,6 +202,16 @@ Documentation for using each command is available by typing:
 > python manage.py help <command>
 ```
 
+### Pickle Tribe gene sets
+
+This step downloads and pickles all the public Tribe gene sets for every
+organism that has been loaded into the database. These pickled files are
+necessary for doing gene set enrichment analyses.
```

Comment:
Well, I thought about that, but notice the instructions in line 83 that talk about having the settings for tribe-client, along with the link to tribe-client. I didn't know whether we should mention that again here.